### PR TITLE
Implement API to create projects

### PR DESCRIPTION
**Pull request by singhabahu**
API endpoint to create projects.

### CODE REVIEW

File: src/api/projects.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+/**
+ * Create project under user's organization
+ * @param  {object} event
+ * @param  {object} context
+ * @param  {object} callback
+ * @return {function} done
+ */
+export const create = (event, context, callback) => {
+  const done = (error, res) => callback(null, {
```
Comment (Rishanthakumar) [inline src/api/projects.js]: @singhabahu  Can't we get this method out and generalize to other apis as well ?

File: src/api/projects.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+  const name = JSON.parse(event.body).name;
+  if (name == null || name.trim() == '') {
+    return done({
+      message: `Request doesn't contain a valid object`,
+    });
+  }
+
+  const uuid = event.requestContext.authorizer.principalId;
+  Permission.hasPermission(uuid, {realm: 'project', action: 'create'})
```
Comment (Rishanthakumar) [inline src/api/projects.js]: @singhabahu cant' we simplify this more ?, l see some readability issues.

**Review by Rishanthakumar — CHANGES_REQUESTED** [review]
Please do the necessary changes.

File: src/api/projects.js (hunk 1 above)
Comment (SavidyaSathyani) [reply src/api/projects.js]: @Rishanthakumar This will be handled in this change.https://github.com/singhabahu/serverless-backend/projects/1#card-10227338

Comment (singhabahu) [reply src/api/projects.js]: @Rishanthakumar please refer to the other PR and confirm it

File: src/api/projects.js (hunk 2 above)
Comment (singhabahu) [reply src/api/projects.js]: any suggestions to improve readability?, this is used to check permission of the user against requested action. any ideas @SavidyaSathyani ?

**Review by Rishanthakumar — APPROVED** [review]

**Pull request merged**